app/src/main/java/com/example/poste/models/Folder.java

```diff
@@ -97,6 +98,25 @@ public void onFailure(Call<ResponseBody> call, Throwable t) {
         }
     }
 
+    /**
+     * Deletes a given post from this folder in the data model and backend
+     *
+     * @param post the post you want to delete
+     */
+    public void deletePost(Post post) {
+        MyApiService apiService = RetrofitClient.getRetrofitInstance().create(MyApiService.class);
+        Call<ResponseBody> call = apiService.deletePost(new DeletePost(post.getId()));
+        call.enqueue(new Callback<ResponseBody>() {
+            @Override
+            public void onResponse(Call<ResponseBody> call, Response<ResponseBody> response) {
+                posts.remove(post);
```

Comment:
Please add a simple `if (response.isSuccessful()) { ... } else { ... } block`. While I understand building towards 'happy day' cases, we want to ensure we aren't needlessly introducing hard-to-troubleshoot bugs (where it appears things were successful, with no logging or display otherwise, even if they were

Reply:
Added base "if (responce.isSuccessful()) {} else {} " branches and Toast messages to provide basic error handling,